tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -348,7 +356,7 @@ def test_01_multiattach_volumes(self):
                 flavor_id=CONF.compute.flavor_ref_alt,
                 user_data=tvaultconf.user_frm_data,
                 key_pair=self.kp,
-                image_id=list(CONF.compute.fvm_image_ref.values())[0])
+                image_id=list(CONF.compute.fvm_image_ref.values())[0], vm_cleanup=True)
```

Comment:
default cleanup value is True, no need to explicitly pass that parameter again.

Reply:
updated